### PR TITLE
Add styling to submit ratings page

### PR DESCRIPTION
**Pull request by jlora23**

### CODE REVIEW

File: src/screens/SubmitRatingScreen.tsx [hunk 1]
```diff
@@ -324,16 +324,29 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
 
 	return (
 		<ScrollView>
-			<PlaceImage photoref={photo_reference} placeName={placeName} />
+			<PlaceImage
+				photoref={photo_reference}
+				placeName={placeName}
+				style={{
+					width: "100%",
+					height: Dimensions.get("window").height * 0.4,
```
Comment (carpenterd777) [inline src/screens/SubmitRatingScreen.tsx:332]:
```suggestion
					height: "40%"
```
did you try this?

File: src/screens/SubmitRatingScreen.tsx [hunk 2]
```diff
@@ -346,48 +359,60 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
...
-				text={S_SUBMIT}
-				accessibilityLabel={S_SUBMIT}
-				onPress={() => void handleSubmitButton(counter, yesNoCounter, placeID)}
-			/>
+			<View
+				style={{
+					alignSelf: "center",
+					marginTop: 10,
+					marginBottom: 30,
+					width: Dimensions.get("window").width * 0.9,
```
Comment (carpenterd777) [inline src/screens/SubmitRatingScreen.tsx:367]:
```suggestion
					width: "90%",
```

File: src/screens/SubmitRatingScreen.tsx [hunk 3]
```diff
@@ -346,48 +359,60 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
...
+		width: Dimensions.get("window").width * 0.9,
+		marginVertical: 12,
+		justifyContent: "space-between",
 	},
 	yesNoRatingOptions: {
 		flexDirection: "row",
-		width: Dimensions.get("window").width,
-		marginTop: 5,
-		marginBottom: 5,
+		width: Dimensions.get("window").width * 0.9,
```
Comment (carpenterd777) [inline src/screens/SubmitRatingScreen.tsx:390]:
```suggestion
		width: "90%",
```

File: src/screens/SubmitRatingScreen.tsx [hunk 4]
```diff
@@ -346,48 +359,60 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
...
+		alignSelf: "center",
+		borderTopColor: "black",
+		borderTopWidth: 3,
 	},
 	popUp: {
-		width: Dimensions.get("window").width * 0.5,
-		height: Dimensions.get("window").height * 0.1,
-		alignSelf: "flex-end",
+		width: Dimensions.get("window").width * 0.45,
+		height: Dimensions.get("window").height * 0.075,
```
Comment (carpenterd777) [inline src/screens/SubmitRatingScreen.tsx:399]:
```suggestion
		width: "45%",
		height: "7.5%",
```

File: src/screens/SubmitRatingScreen.tsx [hunk 5]
```diff
@@ -346,48 +359,60 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
...
-		marginBottom: 5,
-		justifyContent: "space-around",
+		marginBottom: 10,
+		marginHorizontal: 20,
+		justifyContent: "space-between",
 	},
 	placeName: {
 		fontSize: 30,
 		fontWeight: "bold",
+		width: Dimensions.get("window").width * 0.5,
```
Comment (carpenterd777) [inline src/screens/SubmitRatingScreen.tsx:415]:
```suggestion
		width: "50%",
```

**Review by carpenterd777 — CHANGES_REQUESTED** [review]
let's favor using the "X%" relative to the container rather than depending on `Dimension.get` when we can. it might not be possible everywhere, so let us know if you can't somewhere (by leaving a comment).

**Comment by aej11a** [comment]
- Flip voiceover for plus and minus button
- min should be 1
- make the button text smaller with a prop for font size, make the number and title bigger
- change "Cancel" to "Go Back"
- fix checkmarks

**Review by carpenterd777 — APPROVED** [review]

**Pull request merged**